src/MoBi/MoBi.csproj

```diff
@@ -91,4 +91,4 @@
     <Reference Include="Microsoft.VisualBasic" />
     <Reference Include="System.Windows.Forms" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Same for these merge artifacts. Revert where it's only a new single newline line in a project file.